tests/ui/borrowck/copy-suggestion-region-vid.stderr

```diff
@@ -8,6 +8,11 @@ LL |         HelperStruct { helpers, is_empty: helpers[0].is_empty() }
    |                        -------            ^^^^^^^^^^ value borrowed here after move
    |                        |
    |                        value moved here
+   |
+help: consider cloning the value if the performance cost is acceptable
+   |
+LL |         HelperStruct { helpers.clone(), is_empty: helpers[0].is_empty() }
```

Comment:
This suggestion is wrong.

Reply:
emm, need to take care of shorthand, let me fix it.